### PR TITLE
#315 Configurable indentation size.

### LINKED ISSUE #315: Configurable Indentation
WRONG_INDENTATIN rule always uses four indentations. It would be nice if it can be configurable.

### PR DESCRIPTION
**Pull request by portlek**
This pull request closes #315

### CODE REVIEW

**Review by akuleshov7 — COMMENTED** [review]
Awesome, Thank you very much! 
But could you please also update files with configurations like diktat-analysis.yml?

**Review by akuleshov7 — APPROVED** [review]
lgtm

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/indentation/IndentationConfig.kt [hunk 1]
```diff
@@ -20,4 +20,9 @@ internal class IndentationConfig(config: Map<String, String>) : RuleConfiguratio
      * If true, if expression is split by newline after operator like +/-/`*`, then the next line is indented with two indentations instead of one
      */
     val extendedIndentAfterOperators = config["extendedIndentAfterOperators"]?.toBoolean() ?: true
+
+    /**
+     * The indentation size for each file
+     */
+    val indentationSize = config["indentationSize"]?.toInt() ?: 4
```
Comment (akuleshov7) [inline diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/indentation/IndentationConfig.kt:27]: detekt shows a magic number here: 
/home/runner/work/diKTat/diKTat/diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/indentation/IndentationConfig.kt:27:65

**Comment by portlek** [comment]
I guess It's done @akuleshov7 isn't it?

**Comment by akuleshov7** [comment]
> I guess It's done @akuleshov7 isn't it?

yes, thank you!

**Pull request merged**